test/Azure.WebSites.DataProtection.UnitTests/Azure.WebSites.DataProtection.UnitTests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>    
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
This is the only additional change I made apart from the cherry picked commits from main branch for 1ES work (as the tests were failing)